src/views/FamilyData/familyData.js

```diff
@@ -0,0 +1,28 @@
+import React, { useState, useEffect } from 'react';
+import { fetchFamilyData } from '../../services/fetchfamily';
+
+export default function FamilyData() {
+  const [members, setMembers] = useState([]);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const data = await fetchFamilyData();
+
+      setMembers(data);
+    };
+    fetchData();
+  }, []);
+  return (
+    <div>
+      <h1>Family Data</h1>
+      {members.map((member) => (
+        <div key={member.name} {...member}>
+          <h2>{member.name}</h2>
+          <p>Age: {member.age}</p>
+          <p>Hobbies: {member.hobbies}</p>
+          <p>Fun Fact: {member.fun_fact}</p>
+        </div>
```

Comment:
As a refactor, we can consider moving this into its own presentational component.